src/test/java/no/sikt/nva/scrapers/DublinCoreScraperTest.java

```diff
@@ -324,6 +325,21 @@ void nonIsoLanguageShouldBeLoggedAsError() {
         assertThat(record.getErrors(), hasItem(new ErrorDetails(INVALID_LANGUAGE, List.of(nonIsoLanguage))));
     }
 
+    @Test
+    void shouldApplyIsoLanguageOverLanguageWhenBothArePresentAndDoNotErrorIfSameLanguages() {
+        var typeDcValue = new DcValue(Element.TYPE, null, "Journal Article");
+        var peerReviewed = new DcValue(Element.TYPE, null, "Peer reviewed");
+        var isoLanguageDcValue = new DcValue(Element.LANGUAGE, Qualifier.ISO, "ger");
+        var nonIsoLanguageDcValue = new DcValue(Element.LANGUAGE, null, "ger");
```

Comment:
Burde ikke de ha litt ulik språk kode for å sjekke at iso språket blir prioritert? ene med "ger" og den andre med "deutsch"

Reply:
done